backend/test.py

```diff
@@ -158,5 +158,23 @@ def environment_test():
     print(results)
     return results
 
-environment_test()
-chatbot_test()
\ No newline at end of file
+#Delete with a capital D is recognized as a noun and will not work
```

Comment:
can we convert everything to lowercase once we get the input to avoid this?

Reply:
done

Reply:
🤝🤝🤝🤝